sw/device/silicon_creator/lib/manifest.c

```diff
@@ -7,4 +7,7 @@
 // Extern declarations for the inline functions in the manifest header.
 extern rom_error_t manifest_signed_region_get(
     const manifest_t *manifest, manifest_signed_region_t *signed_region);
-extern uintptr_t manifest_entry_point_address_get(const manifest_t *manifest);
+extern rom_error_t manifest_code_region_get(const manifest_t *manifest,
+                                            epmp_region_t *code_region);
+extern uintptr_t manifest_entry_point_get(const manifest_t *manifest,
+                                          uintptr_t *entry_point);
```

Comment:
Return type: `rom_error_t`?
Out-param: `romextimage_entry_point *`?

Reply:
Thanks for catching the typo in the error type! out-param is `uintptr_t` in case we have entry points with different signatures in later boot stages.

Reply:
Fixed the return type.